docs/kafka/kafka-bin-scripts-kafka/README.adoc

```diff
@@ -170,24 +165,23 @@ NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentic
 == Producing messages using Kafka scripts
 
 [role="_abstract"]
-You can use the `kafka-console-producer` script to produce messages to Kafka topics.
+In this task, you use the `kafka-console-producer` script to produce messages to a Kafka topic.
 
 .Prerequisites
 
 * You have a running Kafka instance in {product-kafka}.
```

Comment:
Use the full product name upon first usage in every module.

Reply:
Yes - thanks for catching.